### PR TITLE
Improve test coverage for material.dart

### PR DESCRIPTION
**Pull request by abarth**
Previously these widget had zero test coverage. Also, improve the
coverage tooling Mac.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

**Comment by Hixie** [comment]
LGTM

File: packages/flutter/test/material/chip_test.dart [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                didDeleteChip = true;
+              }
+            )
+          )
+        )
+      )
+    );
+
+    expect(didDeleteChip, isFalse);
+    await tester.tap(find.byType(Tooltip));
```
Comment (eseidel) [inline packages/flutter/test/material/chip_test.dart]: Did you mean Tooltip and not Chip?

Comment (abarth) [inline packages/flutter/test/material/chip_test.dart]: Yeah, the test is tapping on the delete button that's part of the chip.

**Pull request merged**